tests/driver.c

```diff
@@ -217,6 +218,135 @@ void test_receive_mam_message(void) {
   printf("Average time of receive_mam_message: %lf\n", sum / TEST_COUNT);
 }
 
+void test_attach_to_tangle(void) {
```

Comment:
These test functions against IRI proxy APIs are implemented in common. Can you define function template (via macros or helper functions) and then apply the call to APIs accordingly?

Reply:
Copy that!